src/utilities.js

```diff
@@ -0,0 +1,49 @@
+import API from './api.js';
+import Comment from './comment.js';
+
+const api = new API();
+const comment = new Comment();
+
+export default function popUp(e) {
+  const mealID = e.target.dataset.id;
+  const modal = document.getElementById('modal');
+  modal.classList.toggle('hide');
+  modal.innerHTML = '';
```

Comment:
- [ ] Like the previous TSE mentioned, kindly change this and all other `regular functions` into es6 `arrow functions`.